src/cascadia/TerminalSettingsEditor/GlobalAppearance.xaml

```diff
@@ -71,7 +65,16 @@
 
             <!--  Show Titlebar  -->
             <local:SettingContainer x:Uid="Globals_ShowTitlebar">
-                <ToggleSwitch IsOn="{x:Bind ViewModel.ShowTabsInTitlebar, Mode=TwoWay}"
+                <ToggleSwitch x:Name="ShowTitlebarToggleSwitch"
+                              IsOn="{x:Bind ViewModel.ShowTabsInTitlebar, Mode=TwoWay}"
+                              Style="{StaticResource ToggleSwitchInExpanderStyle}"
+                              Toggled="ShowTitleBarToggled" />
+            </local:SettingContainer>
+
+            <!--  Always show tabs  -->
+            <local:SettingContainer x:Uid="Globals_AlwaysShowTabs">
+                <ToggleSwitch x:Name="AlwaysShowTabsToggleSwitch"
+                              IsOn="{x:Bind ViewModel.AlwaysShowTabs, Mode=TwoWay}"

```

Comment:
note to self: we can't just `x:Bind` to `!ViewModel.ShowTabsInTitlebar` here, because we also have to set this value depending on the value of `ShowTabsInTitlebar`.

Reply:
I don't understand your comment. This is an excellent use case for binding, right?

Reply:
oh maybe. There's just the bit where if we turn on `ShowTitlebar`, then we also set `AlwaysShowTabs` to true. Maybe we don't need that, and can just do this with a binding?

Reply:
Nah, you're right that we can't do this with a binding because of the toggling of the other setting.